jobbergate_api/utils.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+
+import click
+
+from jobbergate_api.apps.users.models import User
+from jobbergate_api.apps.users.schemas import pwd_context
+from jobbergate_api.config import settings
+from jobbergate_api.main import db
+
+
+@click.group()
+def cli():
+    pass
+
+
+async def create_super_user(username, email, password):
+    await db.set_bind(settings.DATABASE_URL)
+    await User.create(username=username, email=email, is_active=True, password=password, is_admin=True)
+
+
+@click.option("--password", prompt=True, hide_input=True, confirmation_prompt=True)
+@click.option("--email", prompt=True)
+@click.option("--username", prompt=True)
+@click.command()
+def createsuperuser(username, email, password):
+    password = pwd_context.hash(password)
+    asyncio.run(create_super_user(username, email, password))
+    click.echo(f"User {username} created")
+
+
+cli.add_command(createsuperuser)
+
+if __name__ == "__main__":
```

Comment:
preferred style is just to create an entrypoint so you don't have to make this file executable, and it also makes discoverability of the command better.

then you don't need the last 2 lines at all